tests/abort1.rs

```diff
@@ -130,41 +130,126 @@ fn test_stop_taker_after_setup() {
     taker.recover_from_swap().unwrap();
 
     // ## Fee Tracking and Workflow:
-    //
-    // ### Fee Breakdown:
-    //
-    // +------------------+-------------------------+--------------------------+------------+----------------------------+-------------------+
-    // | Participant      | Amount Received (Sats) | Amount Forwarded (Sats) | Fee (Sats) | Funding Mining Fees (Sats) | Total Fees (Sats) |
-    // +------------------+-------------------------+--------------------------+------------+----------------------------+-------------------+
-    // | Taker            | _                      | 500,000                  | _          | 3,000                      | 3,000             |
-    // | Maker16102       | 500,000                | 463,500                  | 33,500     | 3,000                      | 36,500            |
-    // | Maker6102        | 463,500                | 438,642                  | 21,858     | 3,000                      | 24,858            |
-    // +------------------+-------------------------+--------------------------+------------+----------------------------+-------------------+
-    //
-    //
     // **Taker** => DropConnectionAfterFullSetup
     //
-    // Participants regain their initial funding amounts but incur a total loss of **6,768 sats**
-    // due to mining fees (recovery + initial transaction fees).
+    // Participants regain their initial funding amounts but incur a total loss of **858 sats**
+    // due to mining fees (timelock recovery fee).
     //
     // ### Recovery Fees Breakdown:
-    //
-    // +------------------+------------------------------------+---------------------+--------------------+----------------------------+
-    // | Participant      | Mining Fee for Contract txes (Sats) | Timelock Fee (Sats) | Funding Fee (Sats) | Total Recovery Fees (Sats) |
-    // +------------------+------------------------------------+---------------------+--------------------+----------------------------+
-    // | Taker            | 3,000                              | 768                 | 3,000             | 6,768                      |
-    // | Maker16102       | 3,000                              | 768                 | 3,000             | 6,768                      |
-    // | Maker6102        | 3,000                              | 768                 | 3,000             | 6,768                      |
-    // +------------------+------------------------------------+---------------------+--------------------+----------------------------+
-    //
+    // +------------------+------------------------------------+---------------+
+    // | Participant      |  Timelock Fee (Sats)  | Total Recovery Fees (Sats) |
```

Comment:
No point in adding  `Total Recovery Fee` is its the same. Just have one column with `Timelock Recovery Fee`.

Reply:
Or even better show the full table with other columns empty as you have done for other tests.